wallet/src/ui/app/redux/slices/txresults/index.ts

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    getTransactionDigest,
+    getTransactions,
+    getTransactionKindName,
+    getTransferCoinTransaction,
+    getExecutionStatusType,
+    getTotalGasUsed,
+} from '@mysten/sui.js';
+import {
+    createSlice,
+    createAsyncThunk,
+    createEntityAdapter,
+} from '@reduxjs/toolkit';
+
+import type {
+    TransactionEffectsResponse,
+    GetTxnDigestsResponse,
+    CertifiedTransaction,
+    TransactionKindName,
+    ExecutionStatusType,
+} from '@mysten/sui.js';
+import type { AppThunkConfig } from '_store/thunk-extras';
+
+type TxResultState = {
```

Comment:
We could export this and avoid re-defining it [here](https://github.com/MystenLabs/sui/pull/2529/files#diff-53b58dfabcda4999bb9f97abaf2cfa9f3d528c57631a0dd38c8bd7de06d6af28R14). It's the same right?